CONTRIBUTION.md

```diff
@@ -163,6 +163,7 @@ The new major release is then bundled and the binaries are made available public
 
 The current members of the committee are:
 
+- Walter Binder, Università della Svizzera italiana
```

Comment:
Please do not modify the MD file directly, modify `renaissance-suite.scala` and regenerate the file using `runMain org.renaissance.Launcher --readme`.
Also, please do not use any special char (like `à`) since it doesn't render properly on the website (see my PR #121 )

Reply:
OK, let me fix this once #121 is merged.